src/front/src/pages/home/configuration/CreateConfig.tsx

```diff
@@ -128,15 +128,15 @@ function CreateConfig(props: { id?: number }) {
                 .then(configurationRequest => {
                     if(configurationRequest.succes){
```

Comment:
```suggestion
                    if (configurationRequest.succes) {
```

And why is it ```succes``` ? Shouldn't be ```success``` ?